tests/foreman/ui/test_contentview.py

```diff
@@ -173,6 +173,7 @@ def test_positive_end_to_end(self):
             self.assertIsNotNone(self.content_views.wait_until_element
                                  (common_locators['alert.success']))
 
+    @skip_if_bug_open('bugzilla', 1297308)
     @run_only_on('sat')
     @tier2
     def test_positive_add_puppet_module(self):
```

Comment:
+ 1 @omaciel @sthirugn 
Added an issue #3219  to implement new test that should use  a unique puppet repo/module